content/topics/java-specific/string-and-number-manipulation/_index.md

```diff
@@ -1,3 +1,52 @@
 ---
 title: Strings and numbers in Java
----
\ No newline at end of file
+---
+
+In Java there are multiple ways to manipulate both strings and numbers to work in a manner that is required. These methods can play a vital role in the business logic of a system.
+
+For example: Say you have a list of 100 books and you need to provide the functionality to search for a book. Without being able to manipulate a String
+the user would always have to search for the full title of a book and ensure that every letter's case, matches that of the title.
+
+Java has built in methods to assist with just these kinds of situations.
+
+## Manipulating Strings
+- [Converting](https://www.geeksforgeeks.org/different-ways-for-integer-to-string-conversions-in-java/) variables to Strings.
+
+- [Concatenating Strings](https://www.javatpoint.com/string-concatenation-in-java).
+
+- [Determine the total](https://www.javatpoint.com/java-string-length) number of characters in a String.
+
+- How to alter the [case](https://beginnersbook.com/2013/12/java-string-tolowercase-method-example/) of a String. 
+
+- Finding the [position](https://www.tutorialspoint.com/Difference-between-charAt-and-indexOf-in-Java) of a character in a word/sentence.
+
+- [Comparing](https://www.geeksforgeeks.org/compare-two-strings-in-java/) 2 Strings to check for similarities.
+
+- [Substituting](https://www.javatpoint.com/java-string-replace) new values into a String. 
+
+- [Splitting up](https://www.geeksforgeeks.org/split-string-java-examples/) sentences/words into multiple parts.
+
+- [Remove empty spaces](https://www.journaldev.com/33291/java-string-trim-method-examples) from the start and end of Strings.
+
+
+On the other hand, being able to manipulate numbers is critical to most systems.
+
+## Manipulating Numbers
+- [Converting String](https://www.tutorialspoint.com/Converting-Strings-to-Numbers-and-Vice-Versa-in-Java) values to numbers.
```

Comment:
And for Number class and what function are available you also look into this
https://docs.oracle.com/javase/tutorial/java/data/numberclasses.html